<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRefundVaucher</class>
 <widget class="QDialog" name="DlgRefundVaucher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Refund voucher</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>84</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrint">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>84</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="EQLineEdit" name="leGuest">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="EQLineEdit" name="leCLName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="EQLineEdit" name="leRoom">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="EQLineEdit" name="leCLCode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="EQLineEdit" name="leAmount"/>
   </item>
   <item row="8" column="1">
    <widget class="EQLineEdit" name="lePaymentMode">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Room</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Reservation</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Invoice</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="EQLineEdit" name="lePaymentName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Guest</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="EQLineEdit" name="leInvoice">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Payment mode</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>City ledger</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Voucher</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="EQDateEdit" name="deDate"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="EQLineEdit" name="leVaucher">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="EQLineEdit" name="leReserve">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Balance</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="leBalance">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="rgGuest">
     <property name="text">
      <string>Guest</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QRadioButton" name="rgCL">
     <property name="text">
      <string>City ledger</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">eqdateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
